warehouse/ingest-scripts/src/main/resources/bin/ingest/start-loader.sh

```diff
@@ -121,10 +121,10 @@ if [[${TOTAL} >0 ]]; then
     for ((CUSTOM_LOADER = 0; CUSTOM_LOADER < ${#MAP_LOADER_CUSTOM[@]}; CUSTOM_LOADER = $((CUSTOM_LOADER + 1)))); do
       echo "starting additional map file loader: ${MAP_LOADER_CUSTOM[$CUSTOM_LOADER]}"
       SHUTDOWN_PORT=25100
-      portInUse=$(sudo netstat -tupln | grep $SHUTDOWN_PORT)
+      portInUse=$(ps -eaf | grep [b]ulkIngestMap | grep $SHUTDOWN_PORT)
       while [[ ! -z $portInUse ]]; do
         SHUTDOWN_PORT=$((SHUTDOWN_PORT + 1))
-        portInUse=$(sudo netstat -tupln | grep $SHUTDOWN_PORT)
+        portInUse=$(ps -eaf | grep [b]ulkIngestMap | grep $SHUTDOWN_PORT)
       done
```

Comment:
In hindsight, maybe better to replace the three instances of this logic with a single function(?)

Reply:
Makes sense. Will adjust this in the next.